daemons/attrd/attrd_utils.c

```diff
@@ -103,6 +103,10 @@ attrd_shutdown(int nsig)
         peer_protocol_vers = NULL;
     }
 
+    attrd_unregister_handlers();
```

Comment:
This should be part of attrd_ipc_fini() so there's no change of recreating the table